optimization-utils/linux-optimizations.bash

```
@@ -0,0 +1,351 @@
+# Exit codes
+EXIT_SUCCESS=0
+EXIT_NOT_ROOT=1
+EXIT_UNSUPPORTED_OS=2
+EXIT_COMMAND_FAILED=3
+EXIT_INVALID_ARGS=4
+
+# Default values
+APPLY_CHANGES=false
+VERBOSE=false
+BACKUP=true
+ROLLBACK_ON_ERROR=true
```

Shellcheck (0.10.0)

[warning] ROLLBACK_ON_ERROR appears unused. Verify use (or export if used externally).

(SC2034)